artifacts/manifests/podspec.yaml

```diff
@@ -14,7 +14,7 @@ initContainers:
       memory: 200Mi
     limits:
       cpu: 2
-      memory: 1.6Gi
+      memory: 1.5Gi
```

Comment:
Even though the downward change is small, would it cause the init container to run into OOM errors? How about changing it into an exact value in `Mi`?

Reply:
Exact value in `Mi` = `1536 Mi` right?

Reply:
Let's go with the equivalent of 1.6Gi = 1639Mi.